src/commands/input.rs

```diff
@@ -0,0 +1,439 @@
+//! Implementation of the `input` command.
+
+use std::collections::HashSet;
+
+use anyhow::Result;
+use anyhow::bail;
+use clap::Parser;
+use serde_json::Map;
+use serde_json::Value;
+use wdl::analysis::types::CallKind;
+use wdl::ast::AstNode;
+use wdl::ast::AstToken;
+use wdl::ast::SyntaxKind;
+use wdl::ast::v1::Expr;
+use wdl::ast::v1::InputSection;
+use wdl::ast::v1::LiteralExpr;
+use wdl::ast::v1::Type;
+use wdl::cli::Analysis;
+use wdl::cli::analysis::AnalysisResults;
+use wdl::cli::analysis::Source;
+
+/// Arguments for the `input` subcommand.
+#[derive(Parser, Debug)]
+pub struct InputArgs {
+    /// The path to the WDL document or a directory containing WDL documents to
+    /// validate.
+    #[arg(value_name = "PATH or URL")]
+    pub path: Source,
```

Comment:
How would a directory work? Also this needs a `name` arg to support tasks

Reply:
Good catch. Lazy copy/past inheritance. Task support also needs to be implemented, though it should be fairly easy to add.